packages/unified-pipeline/src/icons/Expand.tsx

```diff
@@ -1,26 +1,14 @@
-import React, { SVGProps } from "react";
+import React, { SVGProps } from 'react'
 
 const Expand = (props: SVGProps<SVGSVGElement>): React.ReactElement => (
-  <svg
-    xmlns="http://www.w3.org/2000/svg"
-    width={24}
-    height={24}
-    fill={props.color}
-    {...props}
-  >
-    <rect
-      width={24}
-      height={24}
-      fill="rgba(147, 147, 159, 0.1)"
-      fillOpacity={0.1}
-      rx={4}
-    />
+  <svg xmlns="http://www.w3.org/2000/svg" width={24} height={24} fill={props.color} {...props}>
```

Comment:
let's move the icons to the `canary/icons` component

Reply:
will do